demo/clean-kubernetes.sh

```diff
@@ -5,14 +5,22 @@ set -aueo pipefail
 # shellcheck disable=SC1091
 source .env
 
+MESH_NAME="${MESH_NAME:-osm}"
+K8S_NAMESPACE="${K8S_NAMESPACE:-osm-system}"
+BOOKBUYER_NAMESPACE="${BOOKBUYER_NAMESPACE:-bookbuyer}"
+BOOKSTORE_NAMESPACE="${BOOKSTORE_NAMESPACE:-bookstore}"
+BOOKTHIEF_NAMESPACE="${BOOKTHIEF_NAMESPACE:-bookthief}"
+BOOKWAREHOUSE_NAMESPACE="${BOOKWAREHOUSE_NAMESPACE:-bookwarehouse}"
```

Comment:
Why is this required? These should already be set when `run-osm-demo.sh` invokes this script?

Reply:
I personally use this script outside of the context of the automated demo, such as the manual demo

Reply:
Got it. Instead of copying the same set of env into each script, set these in your `.env` instead? That's one of the reasons we have `.env`, so different users can customize it based on their use case.